### DISCUSSION TITLE
Throats larger than pores

### DISCUSSION THREAD

**Discussion by kizzle4shaw** [PMEAL/OpenPNM#2734] [Q&A] [answered]
I extracted a network and it seems throats are bigger than pores. How do I fix this??

![image](https://user-images.githubusercontent.com/32852010/231825508-c0f4941b-55e1-4a95-8de2-0837c452bd1f.png)

**Comment by jgostick** [comment]
What does your image look like?  Does it have some anisotropy?  If the pores are flattish, then the throats on the long size can be large, yet the pore body is confined by the smaller dimension.

**Reply by kizzle4shaw** [reply]
It's a cropped image of estaillades carbonate I downloaded it from the Imperial college of London website. Not so sure about anisotropy. I saw on one of the published articles that you supervised on efficient extraction of pore networks from large tomograms you extracted a network from Ketton carbonate which you got from the same source when you first published the snow algorithm. Was this scenario encountered? I know it was 2020 I hope you remember.

**Reply by kizzle4shaw** [reply, marked as answer]
the throats and pores are actually okay, I misinterpreted the graph, incase someone goes through this I thought they should know.
